src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/migration/RemoveRespondentSolOrg.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.migration;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.ORGANISATION_POLICY_RESPONDENT;
+
+public class RemoveRespondentSolOrg {
+
+    public Map<String, Object> migrateCaseData(Map<String, Object> caseData) {
+        caseData.remove(ORGANISATION_POLICY_RESPONDENT);
```

Comment:
can you add a check that the case data has the value and a unit test to cover that